db/schema.rb

```diff
@@ -1375,9 +1377,9 @@
   add_index "instances", ["domain"], name: "index_instances_on_domain", unique: true
 
   create_view "user_ips", sql_definition: <<-SQL
-      SELECT user_id,
-      ip,
-      max(used_at) AS used_at
+      SELECT t0.user_id,
+      t0.ip,
+      max(t0.used_at) AS used_at
```

Comment:
The changes like these in migrations shouldn't be committed, since h they're unrelated